src/components/TimeseriesChart/TimeseriesChart.tsx

```diff
@@ -18,21 +18,25 @@ import { Spin } from 'antd';
 import { decimalTickFormatter, getColor } from '../../utils';
 
 export type TimeseriesChartProps = {
-  panelHeight: number;
+  containerStyle: React.CSSProperties;
```

Comment:
Could we have a prop `styles` like [TenantSelector - Custom Styling](https://github.com/cognitedata/gearbox.js/pull/187/files#diff-14f16035ee023b72b8993bcb65372f41R43) with field `container`?

Reply:
Like this?